tests/utils/integration/test_integrationutils.go

```diff
@@ -32,6 +35,133 @@ import (
+	}
+	for _, validation := range validations {
+		validation()
+	}
+	// Make sure the audit request will work with xsc and not xray
+	assert.NoError(t, os.Setenv(coreutils.ReportUsage, "true"))
+	return func() {
+		assert.NoError(t, os.Setenv(coreutils.ReportUsage, "false"))
+	}
+}
+
+func InitAuditGeneralTests(t *testing.T, minVersion string) {
+	if !*configTests.TestAuditGeneral {
+		t.Skip(getSkipTestMsg("Audit command general integration", "--test.audit"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitAuditJavaScriptTest(t *testing.T, minVersion string) {
+	if !*configTests.TestAuditJavaScript {
+		t.Skip(getSkipTestMsg("Audit command JavaScript technologies (Npm, Pnpm, Yarn) integration", "--test.audit.JavaScript"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitAuditJavaTest(t *testing.T, minVersion string) {
+	if !*configTests.TestAuditJava {
+		t.Skip(getSkipTestMsg("Audit command Java technologies (Maven, Gradle) integration", "--test.audit.Java"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitAuditCTest(t *testing.T, minVersion string) {
+	if !*configTests.TestAuditCTypes {
+		t.Skip(getSkipTestMsg("Audit command C/C++/C# technologies (Nuget/DotNet, Conan) integration", "--test.audit.C"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitAuditGoTest(t *testing.T, minVersion string) {
+	if !*configTests.TestAuditGo {
+		t.Skip(getSkipTestMsg("Audit command Go technologies (GoLang) integration", "--test.audit.Go"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitAuditPythonTest(t *testing.T, minVersion string) {
+	if !*configTests.TestAuditPython {
+		t.Skip(getSkipTestMsg("Audit command Python technologies (Pip, PipEnv, Poetry) integration", "--test.audit.Python"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitScanTest(t *testing.T, minVersion string) {
+	if !*configTests.TestScan {
+		t.Skip(getSkipTestMsg("Other scan commands integration", "--test.scan"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitNativeDockerTest(t *testing.T) (mockCli *coreTests.JfrogCli, cleanUp func()) {
+	if !*configTests.TestDockerScan {
+		t.Skip(getSkipTestMsg("Docker scan command integration (Ubuntu)", "--test.dockerScan"))
+	}
+	return InitTestWithMockCommandOrParams(t, cli.DockerScanMockCommand)
+}
+
+func InitCurationTest(t *testing.T) {
+	if !*configTests.TestCuration {
+		t.Skip(getSkipTestMsg("Curation command integration", "--test.curation"))
+	}
+}
+
+func InitEnrichTest(t *testing.T, minVersion string) {
+	if !*configTests.TestEnrich {
+		t.Skip(getSkipTestMsg("Enrich command integration", "--test.enrich"))
+	}
+	testUtils.ValidateXrayVersion(t, minVersion)
+}
+
+func InitGitTest(t *testing.T) {
+	if !*configTests.TestGit {
+		t.Skip(getSkipTestMsg("Git commands integration", "--test.git"))
+	}
+	err := exec.Command("npm", "install").Run()
```

Comment:
why 'npm install' is an indication for the existence of Git?

Reply:
it should not be here, removed